spec.md

```diff
@@ -164,49 +202,100 @@ echo "SPDX-License-Identifier: MIT" > file.txt
+different directories.
+
+The `version` key (REQUIRED) MUST have an integer value representing the schema
+version of the file. This specification describes version 1 of `REUSE.toml`.
+
+Each `[[annotations]]` table represents an association of Copyright and
+Licensing Information to zero or more Covered Files. It has the following keys:
+
+- `path` (REQUIRED), a string or list of strings representing paths. A path MUST
+  use forward slashes as path separators. A path SHOULD resolve to one or more
+  Covered Files relative to the `REUSE.toml` file's directory. A path MAY use
+  globbing to match several Covered Files in a single expression. These are the
+  globbing and matching rules:
+  - `*` matches everything except forward slashes (i.e. path separators).
+  - `**` and `**/` match everything including forward slashes (i.e. path
+    separators).
+  - In order to escape an asterisk and include it verbatim, prefix it with `\\`.
+    You cannot prefix it with `\` because that is invalid TOML. In order to
+    include a backslash verbatim, use `\\\\`. `\\` followed by any other
+    character is functionally equal to just typing that character.
+- `precedence` (OPTIONAL), a literal string. It determines the order of
+  precedence for Copyright and Licensing Information between the `REUSE.toml` file
+  and Covered Files in the table, and between multiple `REUSE.toml` files if they
+  both contain Information for the same Covered File. Available values are:
+  - `closest`, the default value when `precedence` is not defined. This is an
+    instruction to associate the Copyright and Licensing Information inside of
+    the Covered Files, if available. If no such Information is found, then the
+    Information inside the table of the closest `REUSE.toml` that covers the
+    File is associated. This algorithm is applied separately for copyright and
+    for licensing. If a table for the same File in a closer `REUSE.toml` file
+    has the `override` precedence, then that precedence is applied, and
+    `closest` is ignored. This is effectively a fallback.
+  - `aggregate`. This is an instruction to always associate the Copyright and
+    Licensing Information defined in the table with the table's Covered Files.
+    Subsequently, the `closest` logic is also applied.
+  - `override`. This is an instruction to associate the Copyright and Licensing
+    Information defined in the table with the table's Covered Files, and to
+    ignore any other Information that is closer to the Files. The table in the
+    `REUSE.toml` that is the closest to the root of the Project is
+    authoritative.
+- `SPDX-FileCopyrightText` (OPTIONAL), a string or list of strings. Each string
+  MUST be a Copyright Notice to be associated with the table's Covered Files.
+  The prefix of the Copyright Notice MAY be omitted.
+- `SPDX-License-Identifier` (OPTIONAL), a string or list of strings. Each string
```

Comment:
Shall we define how to interpret a list of license IDs, so that they effectively mean an `AND`?

Reply:
This is kind of in SPDX's ballpark. I'd have to re-read what the SPDX spec says on having multiple such tags.

Reply:
I agree this is part of SPDX, and for SPDX 3.x we should discuss with them how this should be treated, as they got rid of tags in SPDX 3.0.

See also https://github.com/fsfe/reuse-docs/issues/143

Reply:
I think we can get away with not defining this in the REUSE spec.